geometry_msgs/Point[] position
geometry_msgs/Point[] velocity
int32[]            ids
int32[]            tags